src/components/course/EnrollButton.jsx

```diff
@@ -102,20 +104,14 @@ export default function EnrollButton() {
       }
       return (
         <>
-          {isArchived(activeCourseRun) ? (
```

Comment:
B2C does not have this archived check from what I could see in the documentation & code

Reply:
Good catch. All for simplifying this logic anyways 🥇